### PR TITLE
[DOCS][COMMUNITY] add missing header for PMC in community doc

### PR DESCRIPTION
**Pull request by szha**
Thanks for contributing to TVM!   Please refer to guideline https://docs.tvm.ai/contribute/ for useful information and tips. After the pull request is submitted, please request code reviews from [Reviewers](https://github.com/dmlc/tvm/blob/master/CONTRIBUTORS.md#reviewers).

### CODE REVIEW

File: docs/contribute/community.rst [hunk 1]
```diff
@@ -42,9 +47,8 @@ Committers are individuals who are granted the write access to the project. A co
...
 
+Project Management Committee (PMC)
+----------------------------------
 
-Reviewers
----------
-Reviewers are individuals who actively contributed to the project and are willing to participate in the code review of new contributions. We identify reviewers from active contributors. The committers should explicitly solicit reviews from reviewers.  High-quality code reviews prevent technical debt for long-term and are crucial to the success of the project. A pull request to the project has to be reviewed by at least one reviewer in order to be merged.
+The Project Management Committee(PMC) consists group of active committers that moderate the discussion, manage the project release, and proposes new committer/PMC members. Potential candidates are usually proposed via an internal discussion among PMCs, followed by a consensus approval, i.e. least 3 +1 votes, and no vetoes. Any veto must be accompanied by reasoning. PMCs should serve the community by upholding the community practices and guidelines TVM a better community for everyone. PMCs should strive to only nominate new candidates outside of their own organization.
```
Comment (tmoreau89) [inline docs/contribute/community.rst]: Incomplete sentence below:

> PMCs should serve the community by upholding the community practices and guidelines TVM a better community for everyone.

**Comment by tqchen** [comment]
the original community docs puts PMC in the committer section as PMCs are part of the committer  groups. i would recommend to keep the organization as it is.

**Comment by szha** [comment]
@tqchen thanks for clarifying. I thought it was missed since the committer and PMC distinction is common in other places. I will close the PR now.

File: docs/contribute/community.rst (hunk 1 above)
Comment (szha) [reply docs/contribute/community.rst]: Thanks for the review and good catch. I will have someone else open another PR to fix the writing.

**Pull request closed without merging**